src/components/contact/Contact.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from "react";
+
+import ContactMain from "./contactMain/ContactMain";
+import ContactMessage from "./contactMessage/ContactMessage";
+import ContactFAQ from "./contactFAQ/ContactFAQ";
+import {
+  ContactWrapper,
+  HeaderPurpleTransparentLeft,
+  // HeaderPurpleTransparentTop, // <-- plan on using this
+  HeaderImageWrapper,
+  HeaderImage,
+  TitleWrapper,
+  Title,
+} from "./Contact.css";
+
+
+const Contact: FC = () => {
+  return (
+    <>
+      {/* <HeaderPurpleTransparentTop /> */}
+      <HeaderPurpleTransparentLeft />
+      <HeaderImageWrapper id="header-image-wrapper">
+        <HeaderImage id="header-image" />
+      </HeaderImageWrapper>
+      <TitleWrapper>
+        <Title>CONTACT US</Title>
+      </TitleWrapper>
+      <ContactWrapper id="main">
+        <ContactMain />
```

Comment:
This has the “VOLUNTEER" section in it, yes? No need for the "REACH US” section then, unless there’s a design I’m not finding on my end?